specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.3/FormRecognizer.json

```diff
@@ -0,0 +1,2357 @@
+        "modelName": {
+          "$ref": "#/definitions/ModelName"
+        }
+      }
+    },
+    "AnalyzeResult": {
+      "description": "Analyze operation result.",
+      "type": "object",
+      "required": [
+        "version",
+        "readResults"
+      ],
+      "properties": {
+        "version": {
+          "description": "Version of schema used for this result.",
+          "type": "string"
+        },
+        "readResults": {
+          "description": "Text extracted from the input.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ReadResult"
+          }
+        },
+        "pageResults": {
+          "description": "Page-level information extracted from the input.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PageResult"
+          }
+        },
+        "documentResults": {
+          "description": "Document-level information extracted from the input.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DocumentResult"
+          }
+        },
+        "errors": {
+          "description": "List of errors reported during the analyze operation.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ErrorInformation"
+          }
+        }
+      }
+    },
+    "ReadResult": {
+      "description": "Text extracted from a page in the input document.",
+      "type": "object",
+      "required": [
+        "page",
+        "angle",
+        "width",
+        "height",
+        "unit"
+      ],
+      "properties": {
+        "page": {
+          "description": "The 1-based page number in the input document.",
+          "type": "integer",
+          "minimum": 1,
+          "x-nullable": false
+        },
+        "angle": {
+          "description": "The general orientation of the text in clockwise direction, measured in degrees between (-180, 180].",
+          "type": "number",
+          "minimum": -180,
+          "maximum": 180,
+          "exclusiveMinimum": true,
+          "x-nullable": false
+        },
+        "width": {
+          "description": "The width of the image/PDF in pixels/inches, respectively.",
+          "type": "number",
+          "minimum": 0,
+          "x-nullable": false
+        },
+        "height": {
+          "description": "The height of the image/PDF in pixels/inches, respectively.",
+          "type": "number",
+          "minimum": 0,
+          "x-nullable": false
+        },
+        "unit": {
+          "description": "The unit used by the width, height and boundingBox properties. For images, the unit is \"pixel\". For PDF, the unit is \"inch\".",
+          "type": "string",
+          "enum": [
+            "pixel",
+            "inch"
+          ],
+          "x-ms-enum": {
+            "name": "LengthUnit",
+            "modelAsString": false
+          },
+          "x-nullable": false
+        },
+        "language": {
```

Comment:
Please remove the language parameter from the analysis output.  We currently do not output this (except in an edge case due to a bug).

Reply:
done.